src/pybind/mgr/dashboard/frontend/src/app/core/navigation/dashboard-help/dashboard-help.component.html

```diff
@@ -1,9 +1,7 @@
 <form #docsForm
       action="/docs"
       target="_blank"
-      method="post">
-  <input type="hidden"
-         name="token" />
+      method="get">
```

Comment:
Can we get rid of the form (and also ```goToApiDocs()```) and replace it with a link?

Reply:
Done!